src/components/MainContent/TaskModel.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { RenderListItems, Stem, Task } from './index';
+import { Content } from './Content';
+import { MainHeader, NonBulletList, NumberList, Passage, Section, SubHeader } from './Components';
+
+interface TaskModelProps {
+  task: Task;
+  stems: Stem[];
+  evidences: string[];
+  scoringRule: string;
+  index: number;
+}
+
+export const TaskModel = ({ task, stems, evidences, scoringRule, index }: TaskModelProps) => {
+  const taskPrefix = `taskModel${index}`;
+  const desc = Content(task.taskDesc);
+
+  const RenderStemsBy = (target: string, stems: Stem[]) =>
+    stems.filter(stem => stem.shortStem === target).map((stem, index) => {
+      const desc = Content(stem.stemDesc);
+
+      return <li key={index}>{desc}</li>;
+    });
+
+  const RenderAppropriateStems = (stems: Stem[]) => RenderStemsBy('Appropriate Stems', stems);
+
+  const RenderDualTextOnlyStems = (stems: Stem[]) =>
+    RenderStemsBy('Appropriate Stems for Dual-Text Stimuli', stems);
+
+  const RenderFormatExample = (example: string, sectionName: string) => {
+    let elements: JSX.Element | undefined;
+
+    if (example === 'NA') return;
+    if (example === 'Examples') return;
+
+    const parsedExample = Content(example);
+
+    elements = (
+      <Section name={sectionName}>
+        <SubHeader>Format Example</SubHeader>
+        <Passage>{parsedExample}</Passage>
+      </Section>
+    );
+
+    return elements;
+  };
+
+  return (
+    <div>
+      <MainHeader>{task.taskName}</MainHeader>
+      <Section name={`${taskPrefix}-desc`}>
+        <SubHeader>Task Description</SubHeader>
+        <Passage>{desc}</Passage>
+      </Section>
+
+      <Section name={`${taskPrefix}-evidence`}>
+        <SubHeader>Target Evidence Statements</SubHeader>
+        <NumberList>{RenderListItems(evidences)}</NumberList>
```

Comment:
I really like how elegant this component looks with its output, but I think we should still explore the possibility of generating the list items _within_ the `NumberList` and `NonBulletList` components, in order to help us keep style specificity.

Reply:
That would work for NumberList. But NonBulletList requires to work with other rendering functions like RenderStandards, renderAppropriateStems, and renderDualTextOnlyStems. I will change NumberList only. Please let me know if you have better idea.

Reply:
Alright, we'll leave it as it is then.